README.md

```diff
@@ -1,70 +1,92 @@
+> * This is My **Twitter**: [![Twitter Badge](https://img.shields.io/badge/-@mrabdul_ali_-1ca0f1?style=flat-square&labelColor=1ca0f1&logo=twitter&logoColor=white&link=https://twitter.com/mrabdul_ali)](https://twitter.com/mrabdul_ali)&nbsp;
 
-### Analyzing the Bundle Size
+👤 **CHARLOTTE DIVINE**
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+> * This is My **GitHub**: ![GitHub followers](https://img.shields.io/github/followers/divinecharlotte?label=DIVINECHARLOTTE&style=social)
+> * This is My **Linkedin**: [![Linkedin Badge](https://img.shields.io/badge/-DIVINE%20CHARLOTTE-blue?style=flat-square&logo=Linkedin&logoColor=white&link=https://www.linkedin.com/in/charlotte-divine-dusenge/)](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
+> * This is My **Twitter**: [![Twitter Badge](https://img.shields.io/badge/-@divine_maina_-1ca0f1?style=flat-square&labelColor=1ca0f1&logo=twitter&logoColor=white&link=https://twitter.com/divine_maina)](https://twitter.com/divine_maina)&nbsp;
 
-### Making a Progressive Web App
+### BACK END TEAM: 
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+👤 **Natnael Demelash (TEAM LEADER) **
 
-### Advanced Configuration
+- GitHub: [@githubhandle](https://github.com/NatiDeme)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/natnael-demelash/)
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+👤 **Taiwo Damola Adediran**
 
-### Deployment
+- GitHub: [@githubhandle](https://github.com/tylher)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/taiwo-adediran-327654127/)
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
 
-### `npm run build` fails to minify
+## 🤝 Contributing
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ABDULALI3468/bookMe-frontend.git/issues).
+
+## Acknowledgments
+
+- Original design idea by [Murat Korkmaz on Behance](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] On the license section of your readme, the link is broken.  Kindly fix the link.

## Explanation

The link is broken because it points to a file that does not exist.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file, copy it to the root of your project and link to it in your "license" section of your readme.

That way, your app will be well documented 💪